sdcm/cluster.py

```diff
@@ -1098,7 +1098,7 @@ def scylla_setup(self, disks):
         Setup scylla
         :param disks: list of disk names
         """
-        result = self.remoter.run('ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
+        result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
```

Comment:
Is the ip command located in `sbin` on all distributions? why not to use `which ip`?

Reply:
IN debian like distros, ip locates in /sbin directory. IN rhel like distros, ip locates in /usr/sbin, but there is a soft link /sbin which points to /usr/sbin

```
[amos@amos-centos7 ~]$ ls -l / |grep sbin
lrwxrwxrwx    1 root root          8 May 21  2018 sbin -> usr/sbin
```

So /sbin/ip is always effected. However,`which ip`  is better.